README.md

```diff
@@ -1,10 +1,11 @@
 # Mongoose
 
-Mongoose is a [MongoDB](https://www.mongodb.org/) object modeling tool designed to work in an asynchronous environment. Mongoose supports both promises and callbacks.
+Mongoose is a [MongoDB](https://www.mongodb.org/) object modeling tool designed to work in an asynchronous environment. Mongoose supports [Node.js](https://nodejs.org/en/) and [Deno](https://deno.land/) (beta).
 
-[![Slack Status](https://img.shields.io/badge/slack-mongoosejsteam-34D058.svg?logo=slack )](https://mongoosejsteam.slack.com)
 [![Build Status](https://github.com/Automattic/mongoose/workflows/Test/badge.svg)](https://github.com/Automattic/mongoose)
 [![NPM version](https://badge.fury.io/js/mongoose.svg)](http://badge.fury.io/js/mongoose)
+[![Deno version](https://deno.land/badge/mongoose/version)](https://deno.land/x/mongoose)
+[![Deno popularity](https://deno.land/badge/mongoose/popularity)](https://deno.land/x/mongoose)
```

Comment:
i dont know if this is intentional, but the link `https://deno.land/x/mongoose` for me leads to a 404

also `https://deno.land/badge/mongoose/popularity` says `custom badge: resource not found` as the badge message

Reply:
That's expected, we haven't shipped this release so Mongoose isn't on deno.land yet :)